lib/parsers/line_height_parser.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:design_tokens_builder/factory/token_set_factory.dart';
+import 'package:design_tokens_builder/parsers/design_token_parser.dart';
+
+/// Parses the line height according to Design Token Studio.
+///
+/// Supports not only percentage values like 42% but also font sizes like 16.
+///
+/// E.g.
+/// Figma design tokens:
+///   "value": "42%" or
+///   "value": "16"
+///
+/// Flutter generated code:
+///   0.42 or
+///   16
```

Comment:
I've got a problem with this: percentage depends on some other value to be given. Because 50% of 100 is other than 50% of 10. 

The solution may be to create union type like follows:
```dart
class LineHeight {
  factory LineHeight.absolute(double value) = AbsLineHeight(value);
  factory LineHeight.relative(double value) = RelLineHeight(value);
}

class AbsLineHeight {
AbsLineHeight(this.value);
final double value;
}

class RelLineHeight {
RelLineHeight(this.percentage);
final double percentage;
double getValue(double from) => from * percentage;
}
```

It's not pretty. It's not easy to use. But it does not give false impression of the value being the same regardless of what was in the design.

Reply:
Just checked Flutter implementation. They don't even support the absolute line height :D So will remove that and just user percentage based one